src/main/java/uk/gov/justice/probation/courtcaseservice/CreateTableCommandLineRunner.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.probation.courtcaseservice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.*;
+import uk.gov.justice.probation.courtcaseservice.config.DynamoDbConfiguration;
+
+@Component
+@Profile("localDynamo")
+public class CreateTableCommandLineRunner implements CommandLineRunner {
+
+    private final Logger log = LoggerFactory.getLogger(CreateTableCommandLineRunner.class);
```

Comment:
You could use the lombok annotation here, but not sure if this is just a test class with it being a CLI runner